Add article sections clarifying tournament bans and their reach (#6323)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

In addition, all promotional material or any services associated with a tournament receiving official support should adhere to the [osu! community rules](/wiki/Rules). This includes things like Twitter accounts, Discord servers, and so on.

### Integrity

> To ensure officially supported tournaments have the best experience possible, users under an active (or permanent) tournament ban/cooldown are **not** permitted to play in, staff or assist in the running, maintenance or development of said tournament in any way.

Reviewer: ```suggestion
To ensure officially supported tournaments run smoothly, users under an active (or permanent) tournament ban/cooldown are **not** permitted to play in, staff or assist in the running, maintenance or development of such tournaments in any way.
```

> This includes anything from design to commentary, mappooling, refereeing, playtesting and other positions not listed. Put simply, if they are tournament banned, they should not be anywhere near an officially supported tournament in any capacity except as an observer.

Reviewer 2: Would remove design from this list, there's no requirement for designers to be present in a tournament server or even have an osu! account in order to have a product, an unnecessary addition imo.
